src/style.css

```diff
@@ -151,4 +152,431 @@ nav.active a img {
+    padding-top: 2em;
+  }
+
+  .state_title {
+    margin-top: 0;
+    margin-bottom: 0 !important;
+  }
+
+  .section_description {
+    font-size: 15px;
+    font-family: serif;
+  }
+
+  .estates_list {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  .estate {
+    width: 30%;
+    height: 400px;
+    display: flex;
+    flex-direction: column;
+    box-shadow: rgba(100, 100, 111, 0.2) 0 7px 29px 0;
+    align-items: center;
+    margin-bottom: 3em;
+  }
+
+  .estate_image {
+    width: 100%;
+    height: 60%;
+  }
+
+  .details {
+    display: flex;
+    flex-direction: column;
+    padding: 0.5em;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+  }
+
+  .location {
+    display: flex;
+    border-bottom: 2px solid #719e86;
+    padding-top: 1em;
+    padding-bottom: 1em;
+    width: 100%;
+    gap: 20px;
+  }
+
+  .location_icon {
+    color: #719e86;
+  }
+
+  .estate_location {
+    font-size: 15px;
+    color: #6b9b82;
+    font-weight: bold;
+  }
+
+  .estate_details {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+  }
+
+  .detail_line {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .detail_number {
+    font-size: 15px;
+    color: #6b9b82;
+    font-weight: bold;
+  }
+
+  .detail_img_number {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .detail_title {
+    font-size: 12px;
+  }
+
+  .detail_unit {
+    font-weight: bolder;
+    font-size: 7px;
+  }
+
+  .area_detail {
+    display: flex;
+    justify-content: flex-start;
+    gap: 2px;
+  }
+
+  .detail_value {
+    font-size: 15px;
+    color: #6b9b82;
+    font-weight: bold;
+  }
+
+  .estate_price_status {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-top: 1em;
+  }
+
+  .estate_price {
+    font-size: 25px;
+    font-weight: bold;
+  }
+
+  .estate_status {
+    background-color: #6b9b82;
+    border-radius: 20px;
+    display: flex;
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    padding-left: 1em;
+    padding-right: 1em;
+    justify-content: center;
+    align-items: center;
+    color: white;
+  }
+
+  .option_buttons {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    margin-bottom: 3em;
+    gap: 3em;
+  }
+
+  .option_button {
+    background-color: #6b9b82;
+    color: white;
+    border-radius: 20px;
+    font-size: 15px;
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    padding-left: 2em;
+    padding-right: 2em;
+  }
+
+  .option_button:hover {
+    background-color: white !important;
+    color: #6b9b82 !important;
+    box-shadow: #6b9b82 0 7px 29px 0;
+  }
+
+  .general_footer {
+    width: 90%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-image: url(./images/footer_image.jpg);
```

Comment:
<img width="1349" alt="footer" src="https://github.com/Nelitaa/EasyHouse/assets/98427585/941f2b1e-67dc-4264-b28a-fa83a1dc2fde">
The current background image in the footer is great, but have you considered using an image related to real estate with colors that align more closely with the design? This could enhance the thematic consistency of the project.